src/globus_sdk/_testing/data/compute/delete_function.py

```diff
@@ -0,0 +1,13 @@
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+from ._common import FUNCTION_ID
+
+RESPONSES = ResponseSet(
+    metadata={"function_id": FUNCTION_ID},
+    default=RegisteredResponse(
+        service="compute",
+        path=f"/v2/functions/{FUNCTION_ID}",
+        method="DELETE",
+        json={"result": 302},
```

Comment:
I'm curious, what does the 302 in this context mean?

Reply:
It's an HTTP status code integer meant to indicate a successful deletion. This is a relic that we will change in the v3 API route.